packages/@ama-sdk/schematics/schematics/typescript/core/index.ts

```diff
@@ -36,119 +39,184 @@ const getPathObjectTemplate = (pathObj: PathObject) => {
     }`;
 };
 
+const getGlobalProperties = (globalPropertyMap: Map<string, any>, globalProperties: string) => {
+  const globalPropertiesArray = globalProperties.split(',');
+  globalPropertiesArray.forEach((property) => {
+    const [propName, propValue] = property.split('=');
+    globalPropertyMap.set(propName, propValue ?? true);
+  });
+  return globalPropertyMap;
+};
+
+const getGeneratorOptions = (tree: Tree, context: SchematicContext, options: NgGenerateTypescriptSDKCoreSchematicsSchema) => {
+  let openApiToolsJson: any;
+  let openApiToolsJsonGenerator: any;
+
+  // read openapitools.json
+  try {
+    openApiToolsJson = (tree.readJson(path.posix.join(options.directory || '', 'openapitools.json')) as any);
+  } catch {
+    context.logger.warn('No openapitools.json file found in the project');
```

Comment:
```suggestion
    context.logger.warn(`No openapitools.json file found at ${path.posix.join(options.directory || '', 'openapitools.json'} `);
```
you could also extract into a `const openApiToolsPath = path.posix.join(options.directory || '', 'openapitools.json';` before the try catch